[2025-06-28T09:03:49.448+0000] {processor.py:186} INFO - Started process (PID=59) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:03:49.450+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:03:49.454+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:03:49.454+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:03:50.620+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:03:50.618+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:03:50.620+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:03:50.641+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 1.203 seconds
[2025-06-28T09:04:20.696+0000] {processor.py:186} INFO - Started process (PID=74) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:04:20.698+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:04:20.700+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:04:20.700+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:04:21.368+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:04:21.366+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:04:21.368+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:04:21.383+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.693 seconds
[2025-06-28T09:04:52.273+0000] {processor.py:186} INFO - Started process (PID=87) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:04:52.274+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:04:52.276+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:04:52.276+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:04:52.968+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:04:52.966+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:04:52.968+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:04:52.984+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.718 seconds
[2025-06-28T09:05:23.918+0000] {processor.py:186} INFO - Started process (PID=100) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:05:23.919+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:05:23.922+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:05:23.922+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:05:24.609+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:05:24.608+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:05:24.610+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:05:24.625+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.713 seconds
[2025-06-28T09:05:55.572+0000] {processor.py:186} INFO - Started process (PID=120) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:05:55.573+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:05:55.576+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:05:55.575+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:05:56.261+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:05:56.259+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:05:56.261+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:05:56.278+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.713 seconds
[2025-06-28T09:06:26.772+0000] {processor.py:186} INFO - Started process (PID=133) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:06:26.773+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:06:26.950+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:06:26.776+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:06:27.447+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:06:27.445+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:06:27.447+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:06:27.462+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.696 seconds
[2025-06-28T09:06:58.250+0000] {processor.py:186} INFO - Started process (PID=146) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:06:58.253+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:06:58.256+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:06:58.256+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:06:58.771+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:06:58.770+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:06:58.772+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:06:58.791+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.549 seconds
[2025-06-28T09:07:29.585+0000] {processor.py:186} INFO - Started process (PID=159) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:07:29.587+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:07:29.590+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:07:29.589+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:07:30.114+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:07:30.112+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:07:30.114+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:07:30.133+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.555 seconds
[2025-06-28T09:08:00.774+0000] {processor.py:186} INFO - Started process (PID=172) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:08:00.775+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:08:00.778+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:08:00.778+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:08:01.297+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:08:01.296+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:08:01.298+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:08:01.320+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.554 seconds
[2025-06-28T09:08:31.968+0000] {processor.py:186} INFO - Started process (PID=185) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:08:31.970+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:08:31.973+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:08:31.973+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:08:32.700+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:08:32.698+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:08:32.700+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:08:32.727+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.768 seconds
[2025-06-28T09:09:03.703+0000] {processor.py:186} INFO - Started process (PID=198) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:09:03.704+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T09:09:03.708+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:09:03.707+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:09:04.244+0000] {logging_mixin.py:190} INFO - [2025-06-28T09:09:04.242+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T09:09:04.245+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T09:09:04.268+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.573 seconds
[2025-06-28T10:38:57.454+0000] {processor.py:186} INFO - Started process (PID=55) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:38:57.457+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:38:57.460+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:38:57.460+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:38:59.441+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:38:59.440+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:38:59.441+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:38:59.472+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 2.025 seconds
[2025-06-28T10:39:29.635+0000] {processor.py:186} INFO - Started process (PID=75) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:39:29.637+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:39:29.639+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:39:29.639+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:39:30.161+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:39:30.160+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:39:30.161+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:39:30.172+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.541 seconds
[2025-06-28T10:40:00.348+0000] {processor.py:186} INFO - Started process (PID=88) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:40:00.349+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:40:00.351+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:40:00.351+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:40:00.881+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:40:00.880+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:40:00.882+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:40:00.894+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.552 seconds
[2025-06-28T10:40:31.063+0000] {processor.py:186} INFO - Started process (PID=101) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:40:31.064+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:40:31.066+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:40:31.066+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:40:31.578+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:40:31.577+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:40:31.579+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:40:31.591+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.533 seconds
[2025-06-28T10:41:01.744+0000] {processor.py:186} INFO - Started process (PID=114) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:41:01.745+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:41:01.747+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:41:01.747+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:41:02.273+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:41:02.271+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:41:02.273+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:41:02.286+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.547 seconds
[2025-06-28T10:41:33.008+0000] {processor.py:186} INFO - Started process (PID=127) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:41:33.008+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:41:33.010+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:41:33.010+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:41:33.527+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:41:33.526+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:41:33.527+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:41:33.539+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.537 seconds
[2025-06-28T10:42:04.272+0000] {processor.py:186} INFO - Started process (PID=140) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:42:04.274+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:42:04.276+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:42:04.276+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:42:04.799+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:42:04.798+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:42:04.799+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:42:04.812+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.546 seconds
[2025-06-28T10:42:34.960+0000] {processor.py:186} INFO - Started process (PID=153) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:42:34.961+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:42:34.963+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:42:34.963+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:42:35.474+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:42:35.473+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:42:35.475+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:42:35.487+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.532 seconds
[2025-06-28T10:43:05.668+0000] {processor.py:186} INFO - Started process (PID=166) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:43:05.669+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:43:05.671+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:43:05.671+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:43:06.180+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:43:06.179+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:43:06.180+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:43:06.193+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.531 seconds
[2025-06-28T10:43:36.348+0000] {processor.py:186} INFO - Started process (PID=179) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:43:36.348+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:43:36.350+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:43:36.350+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:43:36.724+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:43:36.723+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:43:36.725+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:43:36.740+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.398 seconds
[2025-06-28T10:44:06.900+0000] {processor.py:186} INFO - Started process (PID=192) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:44:06.901+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:44:06.903+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:44:06.903+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:44:07.277+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:44:07.276+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:44:07.277+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:44:07.292+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.398 seconds
[2025-06-28T10:44:37.432+0000] {processor.py:186} INFO - Started process (PID=205) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:44:37.433+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:44:37.435+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:44:37.435+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:44:37.810+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:44:37.809+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:44:37.811+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:44:37.826+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.399 seconds
[2025-06-28T10:45:07.977+0000] {processor.py:186} INFO - Started process (PID=218) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:45:07.979+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:45:07.981+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:45:07.980+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:45:08.365+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:45:08.363+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:45:08.365+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:45:08.378+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.406 seconds
[2025-06-28T10:45:38.529+0000] {processor.py:186} INFO - Started process (PID=231) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:45:38.530+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:45:38.532+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:45:38.532+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:45:38.911+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:45:38.910+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:45:38.911+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:45:38.926+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.402 seconds
[2025-06-28T10:46:09.077+0000] {processor.py:186} INFO - Started process (PID=244) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:46:09.078+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:46:09.080+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:46:09.080+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:46:09.455+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:46:09.453+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:46:09.455+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:46:09.469+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.398 seconds
[2025-06-28T10:46:39.613+0000] {processor.py:186} INFO - Started process (PID=257) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:46:39.614+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:46:39.616+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:46:39.616+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:46:39.990+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:46:39.989+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:46:39.990+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:46:40.005+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.398 seconds
[2025-06-28T10:47:10.156+0000] {processor.py:186} INFO - Started process (PID=270) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:47:10.158+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:47:10.160+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:47:10.159+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:47:10.540+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:47:10.539+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:47:10.541+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:47:10.559+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.408 seconds
[2025-06-28T10:47:40.702+0000] {processor.py:186} INFO - Started process (PID=283) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:47:40.703+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:47:40.705+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:47:40.704+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:47:41.073+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:47:41.072+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:47:41.073+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:47:41.088+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.392 seconds
[2025-06-28T10:48:11.246+0000] {processor.py:186} INFO - Started process (PID=296) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:48:11.247+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:48:11.249+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:48:11.249+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:48:11.633+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:48:11.631+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:48:11.633+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:48:11.651+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.411 seconds
[2025-06-28T10:48:41.795+0000] {processor.py:186} INFO - Started process (PID=309) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:48:41.799+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:48:41.801+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:48:41.801+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:48:42.177+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:48:42.176+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:48:42.177+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:48:42.192+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.402 seconds
[2025-06-28T10:49:12.344+0000] {processor.py:186} INFO - Started process (PID=322) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:49:12.345+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:49:12.347+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:49:12.347+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:49:12.728+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:49:12.727+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:49:12.729+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:49:12.745+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.406 seconds
[2025-06-28T10:49:42.892+0000] {processor.py:186} INFO - Started process (PID=335) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:49:42.927+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:49:42.929+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:49:42.929+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:49:43.307+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:49:43.306+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:49:43.308+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:49:43.321+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.435 seconds
[2025-06-28T10:50:13.464+0000] {processor.py:186} INFO - Started process (PID=348) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:50:13.465+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:50:13.467+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:50:13.466+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:50:13.838+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:50:13.837+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:50:13.838+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:50:13.856+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.398 seconds
[2025-06-28T10:50:44.000+0000] {processor.py:186} INFO - Started process (PID=361) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:50:44.001+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:50:44.004+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:50:44.004+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:50:44.375+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:50:44.373+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:50:44.375+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:50:44.390+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.396 seconds
[2025-06-28T10:51:14.546+0000] {processor.py:186} INFO - Started process (PID=374) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:51:14.547+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:51:14.549+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:51:14.549+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:51:14.925+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:51:14.924+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:51:14.926+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:51:14.941+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.400 seconds
[2025-06-28T10:51:45.090+0000] {processor.py:186} INFO - Started process (PID=387) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:51:45.092+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:51:45.094+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:51:45.093+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:51:45.468+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:51:45.467+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:51:45.469+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:51:45.483+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.399 seconds
[2025-06-28T10:52:15.644+0000] {processor.py:186} INFO - Started process (PID=400) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:52:15.644+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:52:15.646+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:52:15.646+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:52:16.020+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:52:16.019+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:52:16.020+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:52:16.035+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.397 seconds
[2025-06-28T10:52:46.179+0000] {processor.py:186} INFO - Started process (PID=413) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:52:46.181+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:52:46.183+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:52:46.183+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:52:46.558+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:52:46.557+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:52:46.558+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:52:46.573+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.399 seconds
[2025-06-28T10:53:16.721+0000] {processor.py:186} INFO - Started process (PID=426) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:53:16.721+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:53:16.724+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:53:16.723+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:53:17.103+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:53:17.102+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:53:17.104+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:53:17.120+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.405 seconds
[2025-06-28T10:53:47.264+0000] {processor.py:186} INFO - Started process (PID=439) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:53:47.266+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:53:47.268+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:53:47.268+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:53:47.647+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:53:47.646+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:53:47.647+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:53:47.662+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.403 seconds
[2025-06-28T10:54:17.815+0000] {processor.py:186} INFO - Started process (PID=452) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:54:17.816+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:54:17.818+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:54:17.818+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:54:18.197+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:54:18.196+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:54:18.198+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:54:18.214+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.405 seconds
[2025-06-28T10:54:48.361+0000] {processor.py:186} INFO - Started process (PID=466) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:54:48.362+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:54:48.364+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:54:48.364+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:54:48.738+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:54:48.737+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:54:48.739+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:54:48.753+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.398 seconds
[2025-06-28T10:55:18.907+0000] {processor.py:186} INFO - Started process (PID=479) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:55:18.907+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:55:18.909+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:55:18.909+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:55:19.281+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:55:19.280+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:55:19.281+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:55:19.296+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.395 seconds
[2025-06-28T10:55:49.453+0000] {processor.py:186} INFO - Started process (PID=492) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:55:49.454+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:55:49.456+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:55:49.456+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:55:49.837+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:55:49.835+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:55:49.837+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:55:49.852+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.405 seconds
[2025-06-28T10:56:20.006+0000] {processor.py:186} INFO - Started process (PID=505) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:56:20.007+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:56:20.009+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:56:20.009+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:56:20.388+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:56:20.387+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:56:20.389+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:56:20.405+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.404 seconds
[2025-06-28T10:56:50.564+0000] {processor.py:186} INFO - Started process (PID=518) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:56:50.565+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:56:50.568+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:56:50.567+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:56:51.068+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:56:51.066+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:56:51.069+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:56:51.086+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.528 seconds
[2025-06-28T10:57:21.675+0000] {processor.py:186} INFO - Started process (PID=531) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:57:21.676+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:57:21.679+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:57:21.679+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:57:22.074+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:57:22.073+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:57:22.075+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:57:22.089+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.420 seconds
[2025-06-28T10:57:52.274+0000] {processor.py:186} INFO - Started process (PID=544) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:57:52.274+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:57:52.277+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:57:52.276+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:57:52.661+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:57:52.660+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:57:52.661+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:57:52.676+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.410 seconds
[2025-06-28T10:58:22.877+0000] {processor.py:186} INFO - Started process (PID=557) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:58:22.878+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:58:22.880+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:58:22.880+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:58:23.257+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:58:23.255+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:58:23.257+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:58:23.275+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.404 seconds
[2025-06-28T10:58:53.439+0000] {processor.py:186} INFO - Started process (PID=570) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:58:53.440+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:58:53.442+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:58:53.441+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:58:53.828+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:58:53.827+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:58:53.828+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:58:53.843+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.410 seconds
[2025-06-28T10:59:24.013+0000] {processor.py:186} INFO - Started process (PID=583) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:59:24.014+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:59:24.017+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:59:24.016+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:59:24.393+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:59:24.392+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:59:24.394+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:59:24.408+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.402 seconds
[2025-06-28T10:59:54.623+0000] {processor.py:186} INFO - Started process (PID=596) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:59:54.623+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T10:59:54.626+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:59:54.625+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:59:54.997+0000] {logging_mixin.py:190} INFO - [2025-06-28T10:59:54.996+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T10:59:54.998+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T10:59:55.012+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.395 seconds
[2025-06-28T11:00:25.161+0000] {processor.py:186} INFO - Started process (PID=609) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:00:25.162+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:00:25.164+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:00:25.164+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:00:25.545+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:00:25.544+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:00:25.546+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:00:25.561+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.406 seconds
[2025-06-28T11:00:55.704+0000] {processor.py:186} INFO - Started process (PID=622) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:00:55.704+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:00:55.707+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:00:55.706+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:00:56.088+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:00:56.087+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:00:56.088+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:00:56.103+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.406 seconds
[2025-06-28T11:01:26.253+0000] {processor.py:186} INFO - Started process (PID=635) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:01:26.254+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:01:26.256+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:01:26.256+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:01:26.638+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:01:26.637+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:01:26.639+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:01:26.655+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.409 seconds
[2025-06-28T11:01:56.809+0000] {processor.py:186} INFO - Started process (PID=648) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:01:56.810+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:01:56.812+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:01:56.812+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:01:57.217+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:01:57.216+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:01:57.217+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:01:57.232+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.428 seconds
[2025-06-28T11:02:27.384+0000] {processor.py:186} INFO - Started process (PID=661) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:02:27.385+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:02:27.388+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:02:27.387+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:02:27.771+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:02:27.770+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:02:27.772+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:02:27.786+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.408 seconds
[2025-06-28T11:02:57.934+0000] {processor.py:186} INFO - Started process (PID=674) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:02:57.935+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:02:57.937+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:02:57.937+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:02:58.319+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:02:58.318+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:02:58.319+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:02:58.332+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.404 seconds
[2025-06-28T11:03:28.481+0000] {processor.py:186} INFO - Started process (PID=686) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:03:28.484+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:03:28.486+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:03:28.486+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:03:28.861+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:03:28.860+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:03:28.862+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:03:28.878+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.404 seconds
[2025-06-28T11:03:59.037+0000] {processor.py:186} INFO - Started process (PID=699) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:03:59.038+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:03:59.041+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:03:59.040+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:03:59.428+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:03:59.427+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:03:59.428+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:03:59.444+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.412 seconds
[2025-06-28T11:04:29.604+0000] {processor.py:186} INFO - Started process (PID=712) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:04:29.628+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:04:29.631+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:04:29.630+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:04:30.010+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:04:30.009+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:04:30.010+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:04:30.025+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.426 seconds
[2025-06-28T11:05:00.176+0000] {processor.py:186} INFO - Started process (PID=725) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:05:00.177+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:05:00.179+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:05:00.179+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:05:00.560+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:05:00.559+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:05:00.560+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:05:00.577+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.406 seconds
[2025-06-28T11:05:30.727+0000] {processor.py:186} INFO - Started process (PID=738) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:05:30.778+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:05:30.780+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:05:30.780+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:05:31.169+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:05:31.167+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:05:31.169+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:05:31.187+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.465 seconds
[2025-06-28T11:06:01.337+0000] {processor.py:186} INFO - Started process (PID=751) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:06:01.338+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:06:01.340+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:06:01.340+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:06:01.759+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:06:01.757+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:06:01.760+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:06:01.778+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.447 seconds
[2025-06-28T11:06:32.407+0000] {processor.py:186} INFO - Started process (PID=770) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:06:32.408+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:06:32.410+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:06:32.410+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:06:32.792+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:06:32.791+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:06:32.793+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:06:32.806+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.405 seconds
[2025-06-28T11:07:03.009+0000] {processor.py:186} INFO - Started process (PID=783) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:07:03.010+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:07:03.012+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:07:03.012+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:07:03.394+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:07:03.392+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:07:03.394+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:07:03.410+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.406 seconds
[2025-06-28T11:07:33.560+0000] {processor.py:186} INFO - Started process (PID=796) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:07:33.562+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:07:33.564+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:07:33.564+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:07:33.935+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:07:33.934+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:07:33.935+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:07:33.950+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.395 seconds
[2025-06-28T11:08:04.102+0000] {processor.py:186} INFO - Started process (PID=810) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:08:04.103+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:08:04.105+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:08:04.105+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:08:04.479+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:08:04.478+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:08:04.479+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:08:04.497+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.401 seconds
[2025-06-28T11:08:34.645+0000] {processor.py:186} INFO - Started process (PID=823) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:08:34.646+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:08:34.648+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:08:34.648+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:08:35.039+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:08:35.038+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:08:35.039+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:08:35.054+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.416 seconds
[2025-06-28T11:09:05.217+0000] {processor.py:186} INFO - Started process (PID=836) to work on /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:09:05.217+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-06-28T11:09:05.220+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:09:05.220+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:09:05.594+0000] {logging_mixin.py:190} INFO - [2025-06-28T11:09:05.593+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-06-28T11:09:05.595+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-06-28T11:09:05.609+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.398 seconds
